crates/oci-distribution/src/lib.rs

```diff
@@ -215,8 +255,7 @@ mod test {
 
     #[tokio::test]
     async fn test_auth() {
-        let image =
-            Reference::try_from("webassembly.azurecr.io/hello-wasm:v1").expect("parsed reference");
+        let image = Reference::try_from(HELLO_IMAGE).expect("parsed reference");
```

Comment:
Nit: the expect message is what gets printed when unwrapping fails. It should read like "could not parse reference".

Reply:
Fixed